themes/custom/az_barrio/config/install/az_barrio.settings.yml

```diff
@@ -81,7 +81,7 @@ az_hide_front_title: false
 az_back_to_top: true
 az_barrio_navbar_offcanvas: true
 az_barrio_font: true
-az_bootstrap_source: cdn
+az_bootstrap_source: local
 az_bootstrap_cdn_version: stable
```

Comment:
Should we remove this legacy setting and add an update hook to delete it from the active config?

Reply:
I've added the `az_bootstrap_cdn_version` setting back to the form so we can remove it in a later ticket.

Reply:
Added a follow-up issue https://github.com/az-digital/az_quickstart/issues/4674